Demo/PQFCustomLoadersDemo/VBFRotatingSquare.m

```diff
@@ -0,0 +1,264 @@
+//
+//  VBFRotatingSquare.m
+//  VBFFlatLoaders
+//
+//  Created by Victor Baro on 19/09/2014.
+//  Copyright (c) 2014 Victor Baro. All rights reserved.
+//
+
+#import "VBFRotatingSquare.h"
+#import <UIColor+FlatColors/UIColor+FlatColors.h>
+
+
+@interface VBFRotatingSquare ()
+@property (nonatomic, strong) UIView *loaderView;
+@property (nonatomic, strong) UIView *square;
+@property (nonatomic, strong) CAShapeLayer *shadow;
+@property (nonatomic, assign) BOOL animate;
+@property (nonatomic) CGFloat rectSize;
+@end
+
+@implementation VBFRotatingSquare
+
```

Comment:
Missing `- initWithCoder:` and `-drawRect:` for the Inspectable Demo, but I can add them for you later :smile: